sksurv/ensemble/boosting.py

```diff
@@ -473,11 +474,12 @@ class GradientBoostingSurvivalAnalysis(BaseGradientBoosting, SurvivalAnalysisMix
         sparse data will raise an error.
 
     subsample : float, optional, default: 1.0
-        The fraction of samples to be used for fitting the individual base
-        learners. If smaller than 1.0 this results in Stochastic Gradient
+        The fraction of rows to be used for fitting the individual regression
+        trees. If smaller than 1.0, this results in Stochastic Gradient
         Boosting. `subsample` interacts with the parameter `n_estimators`.
         Choosing `subsample < 1.0` leads to a reduction of variance
-        and an increase in bias.
+        and an increase in bias. Lower values improve performance on large 
+        datasets.
```

Comment:
The statement that a lower of value of `subsample` leads to better performance is generally no true. It is a hyper-parameter whose choice depends on the dataset and also other hyper-parameters like n_estimators. Please remove this statement.